konrad/radiation/__init__.py

```diff
@@ -25,6 +25,7 @@
 from .arts import ARTS
 from .radiation import Radiation
 from .rrtmg import RRTMG
+#from .quadrature import quadrature
```

Comment:
Please remove lines that are commented. Commented code tends to become outdated quite easily (as it is never used) and usually only obfuscates what is happening